src/DocumentFormat.OpenXml/SimpleTypes/Int32Value.cs

```diff
@@ -13,6 +14,8 @@ namespace DocumentFormat.OpenXml
     [DebuggerDisplay("{InnerText}")]
     public class Int32Value : OpenXmlComparableSimpleValue<int>
     {
+        private static readonly NumberStyles ParseStyles = NumberStyles.AllowLeadingSign | NumberStyles.AllowLeadingWhite | NumberStyles.AllowTrailingWhite | NumberStyles.AllowExponent;
```

Comment:
Hey @twsouthwick, many thanks for taking the time to look into this!

Just a quick point based on what I noticed trying to work around the issue myself—the number styles you have here won't be able to parse something like 1.000001e6 into 1,000,001. Here's a short fiddle: https://dotnetfiddle.net/EXr0Vw.

To cover that, I think you'll probably have to include `NumberSyles.AllowDecimalPoint` as well, although I'm not sure whether this would have any broader implications in terms of parsing.

Reply:
Oh interesting. Thanks for catching that. Do you see that in the row count?

Reply:
Yes, I do - see the XML sample I added to the issue. It's actually from the very same file that was causing problems.

Reply:
Cool. I've pushed an update for that case.